### ISSUE TITLE
bug: sil_devanagari_typewriter has swapped OSKFont and DisplayFont entries

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyboards#2760]
OK, so looks like @LornaSIL may have inadvertently swapped the OSKFont and DisplayFont entries in PR #2474. Let's treat this as a separate issue to resolve once the 17.0 merge is complete.

_Originally posted by @mcdurdin in https://github.com/keymanapp/keyboards/pull/2759#discussion_r1604358886_

See:

https://github.com/keymanapp/keyboards/pull/2474/files#diff-698fac78eb0c462f56e5152c5dbc2f6b84407069b41ac329a441265c9be9c085L104-R130

**Comment by darcywong00** [comment]
It depends which is correct for OSKFont and DisplayFont

sil_devanagari_typewriter.kps versions:

## 1.1
https://github.com/keymanapp/keyboards/blob/0436e55e211d315b88cd1d580a2d66d8159e4df7/release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps#L115-L119

## 1.1 (current master)
https://github.com/keymanapp/keyboards/blob/9936066591194d4a4dcc9bf9444e927e60b273a8/release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps#L126-L130

## staging-17.0 (fonts matches 1.1, versions 1.1.1)
https://github.com/keymanapp/keyboards/blob/d8cb6228abf1077796bc429c6aff7b511f7f9ec1/release/sil/sil_devanagari_typewriter/source/sil_devanagari_typewriter.kps#L139-L143

I don't know which set stays when staging-17.0 gets merged into master next week...
